From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 21 Apr 2019 00:17:13 +0100
Subject: lockdown: Refer to Debian wiki until manual page exists
Forwarded: not-needed

The lockdown denial log message currently refers to a
"kernel_lockdown.7" manual page, which is supposed to document it.
That manual page hasn't been accepted by the man-pages project and
doesn't even seem to have been submitted yet.  For now, refer to the
Debian wiki.

---
--- a/security/lock_down.c
+++ b/security/lock_down.c
@@ -28,7 +28,7 @@ static void __init lock_kernel_down(cons
 {
 	if (!kernel_locked_down) {
 		kernel_locked_down = true;
-		pr_notice("Kernel is locked down from %s; see man kernel_lockdown.7\n",
+		pr_notice("Kernel is locked down from %s; see https://wiki.debian.org/SecureBoot\n",
 			  where);
 	}
 }
@@ -60,7 +60,7 @@ void __init init_lockdown(void)
 bool __kernel_is_locked_down(const char *what, bool first)
 {
 	if (what && first && kernel_locked_down)
-		pr_notice("Lockdown: %s: %s is restricted; see man kernel_lockdown.7\n",
+		pr_notice("Lockdown: %s: %s is restricted; see https://wiki.debian.org/SecureBoot\n",
 			  current->comm, what);
 	return kernel_locked_down;
 }
